packages/twenty-front/src/modules/ui/input/components/internal/date/components/AbsoluteDatePickerHeader.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from '@emotion/styled';
+import { DateTime } from 'luxon';
+import { IconChevronLeft, IconChevronRight } from 'twenty-ui';
+
+import { LightIconButton } from '@/ui/input/button/components/LightIconButton';
+import { Select } from '@/ui/input/components/Select';
+import { DateTimeInput } from '@/ui/input/components/internal/date/components/DateTimeInput';
+
+import {
+  MONTH_AND_YEAR_DROPDOWN_MONTH_SELECT_ID,
+  MONTH_AND_YEAR_DROPDOWN_YEAR_SELECT_ID,
+} from './InternalDatePicker';
+
+const StyledCustomDatePickerHeader = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
+  padding-left: ${({ theme }) => theme.spacing(2)};
+  padding-right: ${({ theme }) => theme.spacing(2)};
+  padding-top: ${({ theme }) => theme.spacing(2)};
+
+  gap: ${({ theme }) => theme.spacing(1)};
+`;
+
+const months = [
```

Comment:
Avoid harcoding value when you can get them from a lib, maybe something like this?
```
const getMonthLabels = () => {
  return Array.from({ length: 12 }, (_, i) => {
    const date = new Date(2000, i); 
    const monthLabel = new Intl.DateTimeFormat('en-US', { month: 'long' }).format(date);
    return { label: monthLabel, value: i };
  });
};

const months = getMonthLabels();
```